style.css

```diff
@@ -0,0 +1,959 @@
+  color: #ec5242;
+}
+
+#aboutDetail {
+  background-color: white;
+  color: #535353;
+  font-size: 0.85rem;
+  border: 1.5px solid rgb(236, 236, 236);
+  padding: 3%;
+  margin: 6% 0% 2% 0%;
+  text-align: center;
+  line-height: 1.5rem;
+}
+
+#aboutDetail p {
+  margin: 0;
+}
+
+@media (max-width: 767px) {
+  .deskp {
+    display: none;
+  }
+
+  #mobilepart {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #desktop {
+    display: block;
+  }
+
+  #mobilepart {
+    display: block;
+  }
+
+  #topnav {
+    display: block;
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    align-items: center;
+    width: 100%;
+    background-color: rgb(60, 62, 66);
+    gap: 2%;
+  }
+
+  .programs img {
+    width: 20%;
+    object-fit: scale-down;
+    margin-left: 40%;
+    margin-top: 10%;
+  }
+
+  .sponsors img {
+    width: 150px;
+  }
+
+  #topnav img {
+    max-height: 18px;
+  }
+
+  #topnav a {
+    margin-right: 2%;
+  }
+
+  #topnavList {
+    margin: 0%;
+    padding: 1%;
+    list-style: none;
+    display: inline-flex;
+    justify-content: space-evenly;
+    gap: 15%;
+    padding-right: 10%;
+  }
+
+  #topnavList li {
+    font-size: 1.2rem;
+    color: rgb(255, 255, 255);
+  }
+
+  .social {
+    max-height: 20px;
+    max-width: 10rem;
+    color: rgb(255, 255, 255);
+  }
+
+  .dropDown {
+    display: flex;
+    background-color: #fff;
+  }
+
+  .mlogo1 {
+    display: block;
+    text-decoration: none;
+    font-size: 40px;
+    font-weight: 900;
+    color: #742820;
+    cursor: pointer;
+    margin-left: 80px;
+    align-self: center;
+  }
+
+  #menuOpen {
+    display: none;
+  }
+
+  #menuList {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    gap: 2%;
+    align-items: center;
+    list-style: none;
+    padding: 0%;
+    margin: 0%;
+    width: 100%;
+  }
+
+  #menuList a {
+    font-size: 1.2rem;
+    color: #272a31;
+  }
+
+  .campaignBtn {
+    justify-content: end;
+    padding: 1vw;
+    color: #ec5242;
+    background-color: #fff;
+    font-weight: 500;
+    font-size: 15px;
+    margin-left: 40px;
+    border: 5px solid #ec5242;
+    margin-top: 10px;
+    margin-bottom: 5px;
+    margin-right: 8vw;
+  }
+
+  #intro {
+    background-image: url(./Assets/desktopShapes.jpg);
+    padding-top: 10%;
+    padding-left: 15%;
+    padding-right: 25%;
+  }
+
+  #hello {
+    font-size: 4rem;
+  }
+
+  .deskhello {
+    display: none;
+  }
+
+  #autoFest {
+    font-size: 5rem;
+    line-height: 6rem;
+  }
+
+  #eventDetail {
+    font-size: 1.2rem;
+    width: 70%;
+  }
+
+  #eventDetail p {
+    line-height: 2rem;
+  }
+
+  #date {
+    font-size: 2.5rem;
+  }
+
+  #location {
+    font-size: 1.5rem;
+  }
+
+  #mainProgram {
+    font-size: 3rem;
+  }
+
+  #programContainer {
+    display: flex;
+    gap: 3px;
+    width: 90%;
+  }
+
+  .programs {
+    flex-direction: column;
+    width: 20%;
+    padding: 0;
+  }
```

Comment:
- [ ] With the help of the above implemented changes consider using `flex` to align the content of your card properly. As this is going to help create a more responsive and stunning design. To do this kindly consider below suggestions:
    - [ ] Remove the set `width`. After removing it your design should start looking better already.
    - [ ] Add a `display: flex` and `align-items: center`. This should align all the items in the program container to the center.
    - [ ] Remove the `width: 20%` and `margin-left: 40%` you have set on the program image as the align-items property added should help align all your items to the center with any issues.

Reply:
Done